src/extension.ts

```diff
@@ -755,22 +770,23 @@ function loadKubernetesCore(namespace: string | null, value: string) {
     (err) => vscode.window.showErrorMessage(`Error loading document: ${err}`));
 }
 
-function exposeKubernetes() {
-    findKindNameOrPrompt(kuberesources.exposableKinds, 'expose', { nameOptional: false}, (kindName: string) => {
-        if (!kindName) {
-            vscode.window.showErrorMessage('couldn\'t find a relevant type to expose.');
```

Comment:
This error message has been removed. Not sure if it's correct.

Reply:
I believe the removal is correct.  `findKindNameOrPrompt` should return falsy only if we prompted _and_ the user cancelled.  And of course cancellation shouldn't result in a notification.